packages/portal/src/modules/contentful/queries/browse-landing-static-page.graphql

```diff
@@ -225,6 +225,17 @@ query BrowseLandingStaticPage(
               embed
             }
           }
+          ... on PrimaryCallToAction {
+            name
+            text
+            relatedLink {
+              url
+              text
+            }
+            image {
+              ...illustrationFields
```

Comment:
We are not using the image, so consider not requesting it. Once (if) we start using the CallToActionBanner on the landing page we'll use the image.